src/dpr/components/report-list/utils.ts

```diff
@@ -118,8 +118,15 @@ function renderList(
           selectedFilters: FilterUtils.getSelectedFilters(fields, reportQuery, createUrlForParameters),
         }
 
+        const breadcrumb: { text: string }[] = otherOptions?.breadCrumbList as { text: string; href: string }[]
```

Comment:
I'm not sure about this one - there's no guarantee other users will have a breadcrumb list - it's something we ourselves use, but it's in the free-form "otherOptions", rather than being a structured expectation.

Would it be possible to add it to the report list render methods instead? For example, `renderListWithDefinition` has the report definition already, and `renderListWithData` could be changed to have it as an optional parameter.